packages/EasyCfhighlander/templates/code/project.cfhighlander.rb.twig

```diff
@@ -1,3 +1,4 @@
+# AWS ECS Task to update api, worker and cron
```

Comment:
Sorry I didn't see we've written that yesterday... But those components are responsible for generating the CloudFormation templates for the different services: api, worker and cron.

Then each of those CloudFormation templates will create ECS tasks resources yes :)